InstagramStories/Modules/StoryPreview/IGStoryPreviewController.swift

```diff
@@ -41,7 +41,9 @@ final class IGStoryPreviewController: UIViewController, UIGestureRecognizerDeleg
         return gesture
     }()
     
+    private(set) var collectionViewSizeChanged = false
```

Comment:
Documentation Missing. Add some lines of Txt which tells the purpose of these variables

Reply:
done.